### ISSUE TITLE
Domain blocked by Cisco Talos

### ISSUE THREAD

**Issue by mbrookes** [thesephist/unim.press#2]
https://talosintelligence.com/reputation_center/lookup?search=unim.press (There's a link there to dispute it.)

**Comment by pierow2k** [comment]
> In order to submit a dispute ticket you must be logged into your Cisco account. 

Disputation seems to require a Cisco account.  With other similar products, an end-user can dispute the validity of a false-positive report.

**Comment by mbrookes** [comment]
It says you can create a guest account: https://talosintelligence.com/reputation_center/support?dispute=unim.press&type=domain#ticket2

**Comment by pierow2k** [comment]
It doesn't affect my ability to use unim.press.  I was suggesting a solution to your issue.

**Comment by mbrookes** [comment]
It doesn't affect me either. I was just letting Linus know that he _can_ fix it if he chooses, despite your suggestion to the contrary.

**Comment by thesephist** [comment]
Hey all, thanks for chiming in / for letting me know I'm going to consider this out-of-scope as this isn't really an issue with Unimpress per se, just a content blocker. Appreciate the notice regardless.
